src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1ApplicationDomainEventService.kt

```diff
@@ -139,6 +139,9 @@ class Cas1ApplicationDomainEventService(
           eventType = EventType.applicationWithdrawn,
           eventDetails = getApplicationWithdrawn(application, withdrawingUser, eventOccurredAt),
         ),
+        metadata = mapOf(
+          MetaDataName.CAS1_PLACEMENT_APPLICATION_ID to application.id.toString(),
```

Comment:
It's confusing (and we want to change names at some point), but Application and PlacementApplication are different things, so this metadata needs to be captured in the Cas1PlacementApplicationDomainEventService and not here

Reply:
Moved the change to the correct Service.